src/main/java/seedu/modtrek/logic/parser/TagCommandParser.java

```diff
@@ -26,14 +29,25 @@ public TagCommand parse(String args) throws ParseException {
                 ArgumentTokenizer.tokenize(args, PREFIX_TAG);
 
         Code code;
+        boolean isInclude;
 
         try {
-            code = ParserUtil.parseCode(argMultimap.getPreamble());
+            String preamble = argMultimap.getPreamble();
+            String[] preambleParts = preamble.split(" ");
+            code = ParserUtil.parseCode(preambleParts[0]);
+            if (preambleParts[1].toLowerCase(Locale.ROOT).equals("include")) {
+                isInclude = true;
+            } else if (preambleParts[1].toLowerCase(Locale.ROOT).equals("remove")) {
```

Comment:
Oh i realised that you need to add a check within both if statements here in case input only takes in "tag CS1101S" without further arguments, if not there will be an index exception. I'll resolve this in a separate issue.